base_view_inheritance_extension/models/ir_ui_view.py

```diff
@@ -27,11 +27,9 @@ def ast_dict_update(source, update):
         raise TypeError("`update` must be an AST dict")
 
     def ast_key_eq(k1, k2):
-        # python < 3.8 uses ast.Str; python >= 3.8 uses ast.Constant
+        # Python 3.8+ only uses ast.Constant
```

Comment:
You can remove the versioning comment. People should not try to run Odoo 18 with Python 3.8 :smile:

Reply:
Will you remove this comment, or do you think it's still useful?

Reply:
I think it has been removed already, I do not see it in the change.

Reply:
Ah yes, you are right!